deploy/static/deploy.yaml

```diff
@@ -5682,9 +5682,9 @@ spec:
 
     spec:
       containers:
-        - image: timescale/timescale-prometheus:0.1.0-alpha.3.1
+        - image: timescale/promscale:0.1.0-alpha.3.1
```

Comment:
just a comment: isn't the image lagging behind?

Reply:
fixed